### PR TITLE
Reduce using hard code string in eosio-launcher main.cpp, epe184

### PR DESCRIPTION
**Pull request by softprofe**
## Change Description

Start 10271 description ......

EPE 184

Reduce hard code string in eosio-launcher main.cpp.

End 10271 description ......

## Change Type
**Select ONE**
- [ ] Documentation

- [x] Stability bug fix

- [ ] Other

- [ ] Other - special case

## Consensus Changes
- [ ] Consensus Changes

## API Changes
- [ ] API Changes

## Documentation Additions
- [ ] Documentation Additions

### CODE REVIEW

**Review by linhuang-blockone — CHANGES_REQUESTED** [review]
Please use standard PR template and describe your changes.

**Review by heifner — CHANGES_REQUESTED** [review]
This does not appear to address EPE-184 requested changes.

**Comment by softprofe** [comment]
> Please use standard PR template and describe your changes.

Sure

**Comment by softprofe** [comment]
> This does not appear to address EPE-184 requested changes.

File: programs/eosio-launcher/main.cpp [hunk 1]
```diff
@@ -1151,11 +1159,11 @@ launcher_def::write_config_file (tn_node_def &node) {
...
     }
-    cfg << "plugin = eosio::producer_plugin\n";
+    cfg << "plugin = " << producer_plugin_str << "\n";
   }
-  cfg << "plugin = eosio::net_plugin\n";
-  cfg << "plugin = eosio::chain_api_plugin\n"
-      << "plugin = eosio::history_api_plugin\n";
+  cfg << "plugin = " << net_plugin_str << "\n";
+  cfg << "plugin = " << chain_api_plugin_str << "\n"
+      << "plugin = " << history_api_plugin_str << "\n";
```
Comment (linhuang-blockone) [inline programs/eosio-launcher/main.cpp:1166]: I am not sure if EPE 184 just mean to replace a string literal with a variable.

Comment (softprofe) [reply programs/eosio-launcher/main.cpp:1166]: What's your idea then?

**Pull request closed without merging**